### PR TITLE
/people page explanatory text and "related question" button, fixes #991 & fixes #979

### LINKED ISSUE #979: Change caption of Ask a question button in research note
Hi, this is a `first-timers-only issue`. This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!
## The Problem
Take an example research note like https://publiclab.org/notes/liz/10-26-2016/add-reviewers-to-a-language. The caption in the "**Ask a question**" button needs to be changes there since the button links to the question post page with related tags attached to that note. So the Button should show something like "**Ask a related question**" and with a question mark icon beside it instead of an pencil icon. Also the tooltip should show something like "**Ask a question about _topic_**" 

## Solution

See this line https://github.com/publiclab/plots2/blob/master/app/views/sidebar/_related.html.erb#L8. You need to change the title attribute in the `a` tag with the suitable message to change the tooltip. To change the Caption of the button you need to change the locale parameter `ask_a_question` and also change its corresponding Deutsch translation.  You find the locale for the view in https://github.com/publiclab/plots2/tree/master/config/locales/views/sidebar/_related

## Steps to Fix
- [ ] claim this issue with a comment
- [ ] set up a repository locally and make sure that all tests pass
- [ ] Discuss the final caption for the button
- [ ] start a pull request (see contributing to Public Lab software) but mark it as "in progress"
- [ ] fix the issue following the steps above
- [ ] try the manual test listed above
- [ ] make sure all tests pass with rake test
- [ ] alert someone via either the developers list (see below) to have your pull request merged. This may take a few extra steps depending on when you do it, but we'll help you out!

Please email the developers list (see https://publiclab.org/developers) if you have questions!

### LINKED ISSUE #991: Better explanatory text for /people page
This has been marked as a good candidate for becoming a [first-timers-only issue](http://www.firsttimersonly.com) like [these](https://github.com/publiclab/plots2/labels/first-timers-only), meaning that it's simple, self-contained, and with some extra formatting, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider [reformatting or reposting it as a first-timers-only issue](https://publiclab.org/notes/warren/10-31-2016/create-a-welcoming-first-timers-only-issue-to-invite-new-software-contributors), and then ping @jywarren to get it labelled.

****

The template at https://publiclab.org/people currently shows just a brief title and a list of recently active people, and to admins, a notice about moderating users.

The template is here:

https://github.com/publiclab/plots2/blob/master/app/views/users/list.html.erb#L3-L6

Just after that section, we could add a friendly message telling who these people are, and perhaps inviting people to join the site. Here's the text:

> This page shows a list of all Public Lab contributors and is sorted by those who have posted most recently. Click the profile link to view their posting activity.

### PR DESCRIPTION
**Pull request by Pk13055**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue
* [ ] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer
* [x] `schema.rb.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

File: app/views/users/list.html.erb [hunk 1]
```diff
@@ -1,11 +1,14 @@
+<div class='row'>
```
Comment (jywarren) [inline app/views/users/list.html.erb]: Hi, I believe this div has to go inside the `col-md-3` div -- that way it appears inside the column, not above it. Thanks!

**Comment by jywarren** [comment]
Hi, i requested one change, and also, if you can add more explanatory titles to your pull requests, that's very helpful! Also, if you add `fixes #...` with the number of the issue, then when the PR is accepted, it automatically closes the issues. Thanks a lot!

**Comment by Pk13055** [comment]
Thanks for the tips, I shall keep them in mind while submitting future PRs. I moved the requested block into the md-3 column, anything else I need to take another look at?

**Comment by jywarren** [comment]
Hi -- this looks perfect; merging now. If you're looking for a new challenge, please take a look at our `help-wanted` list: https://github.com/publiclab/plots2/labels/help-wanted

Thanks!

**Comment by ebarry** [comment]
Thank you @Pk13055 this is super!!!!!! :1st_place_medal:

**Pull request merged**